src/reuse/download.py

```diff
@@ -40,6 +41,7 @@ def download_license(spdx_identifier: str) -> str:
     """
     # This is fairly naive, but I can't see anything wrong with it.
     url = urljoin(_SPDX_REPOSITORY_BASE_URL, "".join((spdx_identifier, ".txt")))
+    _LOGGER.debug("license from '%s", url)
```

Comment:
missing operant verb 'downloading' :)

otherwise perfect

Reply:
Check, fixed.

Reply:
I just noticed, the single quote isn't closed.